hi-ml-cpath/src/health_cpath/utils/viz_utils.py

```diff
@@ -174,7 +174,7 @@ def plot_heatmap_overlay(
     """
     fig, ax = plt.subplots()
     fig.suptitle(
-        f"{case}: {slide_node.slide_id} P={abs(slide_node.prob_score):.2f} \n Predicted label: {slide_node.pred_label} "
+        f"{case}: {slide_node.slide_id} P={slide_node.pred_prob_score:.2f} \n Predicted label: {slide_node.pred_label} "
```

Comment:
minor: here we repeat the same command 3 times - excellent case for drying and create a function that returns the title

Reply:
Done!